ideas/README.md

```diff
@@ -0,0 +1,141 @@
+# Ideas
+
+## Intro
+
+Welcome to the Google Summer of Code ideas repository!
+Here you can propose new ideas to participate to this fantastic event 
+and contribute to the nix/nixOS/nixpkgs community ecosystem!
+
+Below you will find all instructions to propose your ideas, so don't be shy
+to contribute!
+
+Summerily yours,
+the repo maintainers.
+
+## Submission fields explanation
+
+### ID
+The ID is composed by the current year and an incremental 4 digit integer
+assigned by one of the repo maintainers and will also be the directory name
+that will contain all the artifacts for that idea.
```

Comment:
I think it's just simpler to use numerical IDs instead of having to remember the exact title.

Maybe we can use the PR number associated with the idea as the ID?

Not a huge deal if they think it's not useful, could be my personal preference.